### PR TITLE
Signature

### PR DESCRIPTION
**Pull request by ctlong**

### CODE REVIEW

File: app/views/submissions/index.jbuilder [hunk 1]
```diff
@@ -1,5 +1,6 @@
 json.array! @submissions do |submission|
   json.id submission.id
+  json.signature @submission.signature
```
Comment (h8rry) [inline app/views/submissions/index.jbuilder]: Don't just copy and paste. Read!

File: app/views/submissions/list.jbuilder [hunk 2]
```diff
@@ -7,6 +7,7 @@ json.array! @forms do |form|
   json.contents form.contents
   json.array! form.submissions do |submission|
     json.id submission.id
+    json.signature @submission.signature
```
Comment (h8rry) [inline app/views/submissions/list.jbuilder]: Read

**Comment by h8rry** [comment]
@ctlong please review. the rest looks good.

**Comment by ctlong** [comment]
@ggharry Reviewed and fixed.

**Pull request merged**